tools/netlify.js

```diff
@@ -17,6 +17,8 @@ const SPECS = {
     "https://w3c.github.io/payment-request/",
     "https://w3c.github.io/gamepad/",
     "https://w3c.github.io/hr-time/",
+    "https://w3c.github.io/web-share/",
+    "https://w3c.github.io/trace-context/",
```

Comment:
This uses data-include and markdown extensively.

Reply:
wow... never heard of this spec

Reply:
Any more URLs you would like?

Reply:
Lets get badging in there too.. will suggest